libs/error-analysis/src/lib/ErrorAnalysisDashboard/ColorPalette.ts

```diff
@@ -20,6 +20,19 @@ export class ColorPalette {
   public static ErrorAnalysisDarkBlackText = "rgba(0,0,0,0.8)";
   public static ErrorAnalysisDarkGreyText = "#555";
   public static DisabledColor = theme.palette.neutralSecondary;
+  public static NodeFilledColor = "#F3F2F1";
```

Comment:
Note: per design decision, we want the node shows the same color regardless of themes, so hard-coded the bubble colors